### PR TITLE
Changes for 9.0-staging (no wayland)

### PR DESCRIPTION
**Pull request by rankynbass**
Here's the patches and settings I've been using for my wine builds. These are for wine-staging, not plain wine, so wayland doesn't work, and isn't enabled. You'll need to merge in the changes from upstream before this will work, probably.

Three things to note:

1. On my 8.21 builds, I'm using the _childwindow_fix="true", but that doesn't currently work with 9.0.
2. The "ping-fix.mypatch" implements the fix for Ping Plugin that was posted on discord a while back. It works, but supposedly will be added to upstream wine in the future, at which point it'll have to be removed.
3. The implement-unix-pid-maps-in-winedbg.mypatch has been edited to apply properly. I don't remember exactly what changed, but sometime around 8.12 the original patch stopped working, and had to be adjusted slightly.

### CODE REVIEW

**Comment by marzent** [comment]
Hm regarding 1) do we need Vulkan child windows?
Indeed 2) should be added in the near future, but there is currently a rather large backlog of MRs.
Thanks for rebasing 3), it might actually go upstream someday, although in a different form most likely...

**Comment by rankynbass** [comment]
Regarding 1) probably not. I just enabled a bunch of things to try to fix the stuttering bug after 40 minutes, and that was one of them. Never got around to disabling it.

Regarding 2) & 3), I didn't bother to generate corrected patch files, I literally just opened up the files in a text editor and fixed them. So the meta-data is probably not correct, but for the purpose of patching, it's fine.

File: wine-tkg-git/customization.cfg [hunk 1]
```diff
@@ -16,7 +16,7 @@ _nomakepkg_prefix_path=""
 _nomakepkg_midbuild_prompt="false"
 
 # Set to the distro of your choice to attempt dependency resolution. Valid options are "debuntu" (for debian, ubuntu and similar), "fedora" or "archlinux".
-_nomakepkg_dep_resolution_distro=""
+_nomakepkg_dep_resolution_distro="debuntu"
```
Comment (marzent) [inline wine-tkg-git/customization.cfg:19]: will this not blow up the arch and fedora builds?

**Comment by rankynbass** [comment]
Oops. Yeah, I'll fix that. And now that you mention it, we might want to switch all builds over to wow64 instead of building 32 bit libs.

**Comment by marzent** [comment]
Yeah, let's do that; should make the dependency situation a lot easier

**Comment by rankynbass** [comment]
Updated with the fixes. No longer has debuntu, and sets _NOLIB32="wow64"

**Review by marzent — APPROVED** [review]

**Pull request merged**